src/pybind/mgr/dashboard/tox.ini

```diff
@@ -62,22 +66,18 @@ addopts =
     --rcfile=.pylintrc
     --jobs={[pylint]jobs}
 
-[rstlint]
-dirs =
-    README.rst
-    HACKING.rst
-
 [testenv:lint]
 basepython=python3
 deps =
     -rrequirements.txt
     -cconstraints.txt
     -rrequirements-lint.txt
+whitelist_externals = *
```

Comment:
I agree with your suggestion: the initial idea was to avoid updating constantly this line, but we can ssume the environemnt shouldn't be updated very frequently.